_data/simple-icons.json

```diff
@@ -4147,6 +4147,11 @@
             "hex": "0063DC",
             "source": "https://www.flickr.com"
         },
+        {
+            "title": "Flightaware",
+            "hex": "19315B",
```

Comment:
Where did you pull this hex from?
Brand guidelines would suggest either of these two:
![image](https://github.com/simple-icons/simple-icons/assets/1830867/da054b5b-0d00-4904-bf75-22f7cc56f486)